scripts/background.js

```diff
@@ -373,13 +373,13 @@ chrome.tabs.onUpdated.addListener(function (tabId, info) {
           }
           if (event1.auto_update_context === true) {
             if (event1.show_context === "tab") {
-              if (contexts.findIndex(e => e.tab !== 0) >= 0 || windowIdtest !== 0) {
+              if ((contexts.findIndex(e => e.tab !== 0) >= 0 || windowIdtest !== 0)) {
```

Comment:
The extra parenthesis shouldn't be necessary

Reply:
Thanks @MaxReinisch 
That is fixed now